go/telemetry.go

```diff
@@ -0,0 +1,49 @@
+package observe
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+// This is a shared type for a span or trace id.
+// It's represented by 2 uint64s and can be transformed
+// to different string or int representations where needed.
+type TelemetryId struct {
```

Comment:
There wasn't enough bytes in this object to generate a true trace id. Instead of adding a package like uint128 I just use 2 uint64s here.

Reply:
thank you! good idea